File: app/src/main/java/com/example/myapp/DBService/RequestForAll.java
package com.example.myapp.DBService;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class RequestForAll {
    public String name;
    public String country;
    public long time;
    public float temperature;
    public boolean favorite;

    public String getTimeString() {
        Date date = new Date(time);
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("HH:mm", Locale.getDefault());
        return simpleDateFormat.format(date);
    }

    public String getDateString() {
        Date date = new Date(time);
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd.MM.YY", Locale.getDefault());
        return simpleDateFormat.format(date);
    }

    public String getTemperatureString() {
        return String.format(Locale.getDefault(), "%+.0f°C", temperature);
    }

}

File: app/src/main/java/com/chplalex/shaman/mvp/model/db/RequestForAll.kt
package com.chplalex.shaman.mvp.model.db

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

class RequestForAll {

    var name: String = ""
    var country: String = ""
    var time: Long = 0
    var temperature = 0f
    var favorite = false
    val timeString: String
        get() {
            val date = Date(time)
            val simpleDateFormat = SimpleDateFormat("HH:mm", Locale.getDefault())
            return simpleDateFormat.format(date)
        }
    val dateString: String
        get() {
            val date = Date(time)
            val simpleDateFormat = SimpleDateFormat("dd.MM.yy", Locale.getDefault())
            return simpleDateFormat.format(date)
        }
    val temperatureString: String
        get() = String.format(Locale.getDefault(), "%+.0f°C", temperature)
}